README.md

```diff
@@ -90,47 +90,77 @@ Members have different degrees of love for each of the communities that they par
+
+| Member Name       | Activity Type             | Date       | Weight |
 | ----------------- | ------------------------- | ---------- | ----- |
-| Jennifer Lewis    | Attended our conference   | 2019-11-14 | 5     |
-| Terrence Fletcher | Opened a pull request     | 2019-12-21 | 4     |
-| Marie Monzo       | Answered a forum question | 2020-01-14 | 2     |
-| Jennifer Lewis    | Spoke at a meetup         | 2020-02-11 | 7     |
+| Jennifer Lewis    | Attended our conference   | 2019-11-14 | 1     |
+| Terrence Fletcher | Opened a pull request     | 2019-12-21 | 1     |
+| Marie Monzo       | Answered a forum question | 2020-01-14 | 3     |
+| Marie Monzo       | Had a pull request merged | 2020-01-20 | 3     |
+| Jennifer Lewis    | Spoke at a meetup         | 2020-02-11 | 3     |
+
+Once we have a member's activity history, we look at the weight of each activity along with how many months ago it ocurred to arrive at the value for each individual activity. 
+
+`value(month) =  (sum of activity values) * (0.9 ^ number of months ago)`
+
+For activities completed during the current month, `number of months ago` = 0.
+
+To your table, add columns for "Months ago" and "Value." 
+
+In this example, let's say today is February 20, 2020, so February activites are zero months ago, January activities are 1 month ago, etc. 
 
-Once we have a member's activity history, we look at both the _maximum_ and _sum_ of the activity scores. The **maximum** tells us the largest contribution the member has made, and the **sum** gives us a picture of their overall participation level.
+| Member Name       | Activity Type             | Date       | Weight| Months ago | Value |
+| ----------------- | ------------------------- | ---------- | ----- | ---------- | ----- |
+| Jennifer Lewis    | Attended our conference   | 2019-11-14 | 1     | 3          | 0.729 |
+| Terrence Fletcher | Opened a pull request     | 2019-12-21 | 1     | 2          | 0.81  |
+| Marie Monzo       | Answered a forum question | 2020-01-14 | 3     | 1          | 2.7   |
+| Marie Monzo       | Had a pull request merged | 2020-01-20 | 3     | 1          | 2.7   |
+| Jennifer Lewis    | Spoke at a meetup         | 2020-02-11 | 3     | 0          | 3.0   |
 
-| Member Name       | Max Score | Total Score | Love (= Max Score) |
-| ----------------- | --------- | ----------- | ------------------ |
-| Jennifer Lewis    | 7         | 12          | 7                  |
-| Terrence Fletcher | 4         | 4           | 4                  |
-| Marie Monzo       | 2         | 2           | 2                  |
 
-It’s important to track both the sum and the max, but the **max** tells us more about the most impactful contribution the member can make, and helps determine which opportunity to offer next. More on that below.
+Then, to calculate love, we sum the values for each member over the past 12 months. 
```

Comment:
I’m not 100% sure about that, but it looks like the Orbit app doesn’t stop at the last 12 months when computing love. Do we want a cutoff at 12 months, even with the decay built into the formula?

Reply:
The app goes all the way back currently yeah. I think we could leave this out for now in the spirit of less is more, and knowing it might change.

Reply:
See if the update works.

Reply:
works like a charm!